src/config/services/index.md

```diff
@@ -32,3 +37,63 @@ Optionally, a service directory may contain:
 
 When a new service is created, a `supervise` folder will be automatically
 created on the first run.
+
+### Configuring services
+
+Most services can take configuration options set by a `conf` file in the service
+directory. This allows service customization without modifying the service
+directory provided by the relevant package.
+
+Check the service file for how to pass configuration parameters. Services
+usually expect `OPTS="--value ..."` in the `conf` file.
+
+To make more complex customizations than provided by default, [edit the
+service](#editing-services).
+
+### Editing services
+
+To edit a service, first copy its service directory to a different directory
+name, otherwise [xbps-install(1)](https://man.voidlinux.org/xbps-install.1) can
+overwrite the service directory. Then, edit the new service file as needed.
+Finally, the old service should be stopped and disabled, and the new one should
+be started.
+
+## Logging
+
+### Syslog
+
+The default installation comes with no syslog daemon. However, there are syslog
+implementations available in the Void repositories.
+
+#### Socklog
+
+[socklog(8)](https://man.voidlinux.org/socklog.8) is a syslog implementation
+from the author of [runit(8)](https://man.voidlinux.org/runit.8). Use socklog if
+you're not sure which syslog implementation to use. To use it, install the
+`socklog-void` package, and enable the `socklog-unix` and `nanoklogd` services.
+
+The logs are saved in sub-directories of `/var/log/socklog/`, and `svlogtail`
+can be used to help access them conveniently.
+
+The ability to reading logs is limited to `root` and users who are part of the
+`socklog` group.
+
+#### Other syslog daemons
+
+The Void repositories also include packages for `rsyslog` and `metalog`.
+
+## Console
+
+### Disabling default ttys
+
+Void enables [agetty(8)](https://man.voidlinux.org/agetty.8) services for `ttys`
+1 to 6 by default.
+
+To disable agetty services, remove the service symlink and create a `down` file
+in the agetty service directory to avoid that updates of the `runit-void`
+package re-enable the service.
+
+```
+# unlink /var/service/agetty-tty6
```

Comment:
For my own edification, why is it better to use `unlink` rather than `rm` to disable services?

Reply:
This is actually something I wanted other's opinions on. I prefer using unlink because the action is more obvious and I think for documentation purposes it's more explicit.

Reply:
That's a good point. i, too, would be interested if there's any reason(s) to *not* use `unlink`.

Reply:
Duncaen recommended leaving it as `rm` :p